src/app/app.component.css

```diff
@@ -0,0 +1,29 @@
+.side-bar {
+  background-color: gray;
+  height: 100vh;
+}
+
+.name {
+  padding-top: 15px;
+  text-align: center;
+  color: black;
+  font-size: 30pt;
+  font-weight: bold;
+  font-family: 'Garamond';
+}
+
+.work-experience {
+  padding-top: 70px;
+  text-align: center;
+  color: white;
+  font-size: 14pt;
+  font-family: 'Futura', 'Trajan', 'sans-serif';
+}
+
+.education {
+  padding-top: 25px;
+  text-align: center;
+  color: white;
+  font-size: 14pt;
+  font-family: 'Futura', 'Trajan', 'sans-serif';
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Unquote this generic font family name. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Aquoted-generic-font-family-names)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a generic font family at the end of the declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=css%3Afont-family-not-ending-with-generic-font-family)